test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -18,18 +76,37 @@ namespace Test::KozaniManagerRuntimeTests
         TEST_CLASS_SETUP(ClassSetup)
         {
             ::TB::Setup();
+            ::TP::AddPackage(KozaniManagerTestPackage::c_PackageDirName, KozaniManagerTestPackage::c_PackageFullName);
             return true;
         }
 
         TEST_CLASS_CLEANUP(ClassCleanup)
         {
+            ::TP::RemovePackageIfNecessary(KozaniManagerTestPackage::c_PackageFullName);
             ::TB::Cleanup();
             return true;
         }
 
-        // TODO Replace this with your own tests
-        TEST_METHOD(JustDoIt)
+        TEST_METHOD(ActivateRemoteApplicationTests)
         {
+            try
+            {
+                winrt::Microsoft::Kozani::ManagerRuntime::ManagerRuntimeManager runtimeManager;
+                auto statusCallback{ winrt::make_self<MyKozaniStatusCallback>() };
+
+                runtimeManager.ActivateRemoteApplication(winrt::Windows::ApplicationModel::Activation::ActivationKind::Launch,
+                    L"Microsoft.WindowsCalculator_8wekyb3d8bbwe!App", L"c:\\data\\connection.rdp", L"c:\\data\\additionalSettings.txt", 
+                    nullptr,    // IActivatedEventArgs
+                    statusCallback.as<winrt::Windows::Foundation::IInspectable>());
+
+                VERIFY_IS_TRUE(statusCallback->IsActivated(), L"IKozaniStausCallback::OnActivated() should have been called.");
+            }
+            catch (const winrt::hresult_error& e)
+            {
+                VERIFY_FAIL(WEX::Common::String().Format(
+                    L"ActivateRemoteApplication() failed with 0x%x, error message: %s",
+                    e.code(), e.message().c_str()));
```

Comment:
If you don't catch it what does TAEF output? Usually it's message is meaningful enough, doing your own catch/printf is redundant

Reply:
VERIFY_FAIL will fail the test. TAEF will properly handle it and mark the test as failed.

Reply:
A thrown exception boiling up out of your test method is caught by TAEF and fails the test. All properly handled, no need for you to add your own catch block redundant with TAEF's behavior we get for free

Reply:
Ok, removed try catch.